pwsh_module/command.tests.ps1

```diff
@@ -5,6 +5,10 @@ BeforeAll {
     return "bolt " + $params -join " "
   }
 
+  Mock -ModuleName 'PuppetBolt' -Verifiable -CommandName Get-BoltVersion -MockWith {
+    return "3.0.0"
```

Comment:
Are we going to need to update this every time we release, or can it be mocked?

Reply:
Thanks for catching this. This was needed in a prior version of the PR, but is not needed with the final version of the `Get-BoltVersion` function. Removing this now

If this were staying in, it wouldn't matter that the version was hardcoded because the tests were testing parameter presence, not version correctness.